scalyr_agent/third_party/licenses/LICENSE_activestate.txt

```diff
@@ -19,7 +19,7 @@ ACTIVESTATE SOFTWARE INC. ("ACTIVESTATE") IS WILLING TO LICENSE THE SOFTWARE ONL
    a. You are granted worldwide, perpetual, paid up, royalty free, non-exclusive rights to install and use the Software subject to the terms and conditions contained herein.
 
 
-   b. You may: (i) copy the Software for archival purposes, (ii) copy the Software for personal use purposes, (iii) use, copy, and distribute the Software solely for Your organization's internal non-production use and or internal business operation purposes including copying the Software to other workstations inside Your organization, (iv) redistribute parts of the Software outside of Your organization only as part of a Wrapped Application utilizing executable generators such as PerlApp, Perl2Exe, PAR, TclApp, py2app, or py2exe. Any copy must contain the original Software's proprietary notices in unaltered form. "Wrapped Application" means a single-file executable wherein all binary components are encapsulated in a single binary however You may not expose the base programming language as a scripting language within your own application program to end users.
+   b. You may: (i) copy the Software for archival purposes, (ii) copy the Software for personal use purposes, (iii) use, copy, and distribute the Software solely for Your organization's internal non-production use and or internal business operation purposes including copying the Software to other workstations inside Your organization, (iv) redistribute parts of the Software outside of Your organization only as part of a Wrapped Application utilizing executable generators such as PerlApp, Perl2Exe, PAR, TclApp, py2app, py2exe or PyInstaller. Any copy must contain the original Software's proprietary notices in unaltered form. "Wrapped Application" means a single-file executable wherein all binary components are encapsulated in a single binary however You may not expose the base programming language as a scripting language within your own application program to end users.
```

Comment:
I don't think we should be changing the text for another library's LICENSE.  We should rever this change.  After all, they do say you cannot alter the notices in any form.

In practice, we don't need to have PyInstaller listed explicitly here anyway because they say "executable generators such as " -- it is not meant to be an exhaustive list.